### PR TITLE
Time Void to Time Lab

### PR DESCRIPTION
**Pull request by ValkyrieSkies**
Now you too can experience the tense last moments as elite Time Agent John Beckett departs from his lab in the ruined, apocalyptic future on his mission to save the timeline, rather than picking up from some strange interim room with children's carpet he must have wound up in by accident along the way.

timelab

## Changelog

:cl:
 * tweak: Remaps the Time Void to actually be something

### CODE REVIEW

File: maps/misc/timevoid.dmm [hunk 1]
```diff
@@ -3,21 +3,350 @@
...
+/turf/simulated/floor/dark,
+/area/timevoid)
+"j" = (
+/obj/machinery/disposal/compactor,
+/turf/simulated/floor/dark,
+/area/timevoid)
+"k" = (
+/obj/structure/table/reinforced,
+/obj/machinery/computer/fluff/terminal/old{
+	desc = "A worn and well used computer. It appears to be running Gentoo."
```
Comment (Kurfursten) [inline maps/misc/timevoid.dmm]: >not custom-named as "IBM 5100"

Comment (ValkyrieSkies) [reply maps/misc/timevoid.dmm]: FUCK

Comment (ValkyrieSkies) [reply maps/misc/timevoid.dmm]: Addressed and improved further.

**Comment by ExpulsionSnooper** [comment]
cool but doesn't align with muh canon

**Comment by ValkyrieSkies** [comment]
> cool but doesn't align with muh canon

Massive skill issue, simply ignore it and roleplay your canon.

**Comment by ExpulsionSnooper** [comment]
when i said muh canon i meant how the situation is described in the wiki/some flavor text, so if you're gonna make this change you better be prepared to update more than just the map

**Comment by ExpulsionSnooper** [comment]
Actually, now that I think about it, him having a lab almost completely contradicts the current lore/explanation of him finally discovering time travel, being abducted by the time agency, and force to go on missions to fix/repair the timeline for all of eternity. I could also probably go into how this would actually also be a different type of time travel then what is currently shown, but I'm sure you don't want to hear it.
t. timetravel autist

**Comment by Eneocho** [comment]
I made the wiki page based on the merged time agent PR, with all the flavour inspited by said PR.
I like the lore of John being essentially trapped in the time tampering loop.
However, what if the entire time agency is different John Becketts from different points in time? The founder was the first john to find out about time travel, and all "new" johns are sent on missions that are impossible without time tampering so they can separate the johna that got lucky from the johns with true understanding. Instead of always being the same john, its always a new john, and the evil twin is actually another agent sent to test the first.

**Comment by Eneocho** [comment]
Change it from time lab to time hub, where all Johns must go before they leap timelines (think the end of time in chrono trigger).

**Comment by ExpulsionSnooper** [comment]
Sure, but I kinda liked how vague it was, and the GRIMDARK element of some guy needing to do infinite missions, but I wouldn't mind it being what you described, like a small wing of the time agency with some new sprites for a time machine and some futuristic stuff in there.

**Comment by ValkyrieSkies** [comment]
Keep in mind that the text of my PR does not actually mean there's going to be a change to the lore, it's merely a flavourful amble into the PR. The timevoid could easily be a time agency outpost manned by John or whatever, and the paper is just a dumb excuse to make an obnoxious amount of time travel references.



> Change it from time lab to time hub, where all Johns must go before they leap timelines (think the end of time in chrono trigger).

I didn't actually change the name, it's still "Time Void" as far as the game is concerned.

**Comment by BomberBro** [comment]
\>ss13
\>lore
shiggydiggy

**Comment by ExpulsionSnooper** [comment]
I just think it's interesting, okay

**Comment by ShiftyRail** [comment]
@BomberBro I'm gonna steal your OC for this meme.

**Pull request merged**